byzcoin/bcadmin/cmd_db.go

```diff
@@ -525,12 +546,22 @@ func dbCheck(c *cli.Context) error {
 					errStrFl)
 				continue
 			}
-			err := fl.VerifyWithScheme(pairing.NewSuiteBn256(),
-				sb.Roster.ServicePublics(skipchain.ServiceName),
-				sb.SignatureScheme)
-			if err != nil {
-				log.Errorf("%s fails signature verification: %+v",
-					errStrFl, err)
+			if len(next.BackLinkIDs) <= i {
+				log.Errorf("%s points to a block with too few backlinks", errStrFl)
+				continue
+			}
+			if !next.BackLinkIDs[i].Equal(sb.Hash) {
```

Comment:
too late for that, but this code doesn't breath. Each block of `if` is like a different paragraph of an essay. It needs blanks lines to help the reader understand and rhythm the story.

Reply:
I have a personal preference of putting similar `if` blocks together, else it get's too 'spacy'. But you're right, it needed some more empty lines... Is this better?